common/display/src/main/scala/uk/ac/wellcome/display/models/v2/DisplayCollection.scala

```diff
@@ -1,23 +1,57 @@
-package uk.ac.wellcome.display.models.v2
+package uk.ac.wellcome.display.models
 
-import io.circe.generic.extras.JsonKey
 import io.swagger.v3.oas.annotations.media.Schema
-import uk.ac.wellcome.models.work.internal.Collection
+
+import uk.ac.wellcome.models.work.internal._
+import uk.ac.wellcome.display.models.v2.DisplayWorkV2
 
 @Schema(
   name = "Collection",
-  description = "A collection that a work is part of"
```

Comment:
It's still the collection this work is part of.

Reply:
See this comment: https://github.com/wellcomecollection/catalogue/pull/499#discussion_r398643348

Happy for either description /  a new one if suggested.